packages/fetch/src/index.ts

```diff
@@ -104,15 +104,19 @@ ${
   }
 }\n`;
 
+  const isNdJson = response.contentTypes.some(
+    (c) => c === 'application/nd-json' || c === 'application/x-ndjson',
+  );
   const responseTypeName = fetchResponseTypeName(
     override.fetch.includeHttpResponseReturnType,
-    response.definition.success,
+    isNdJson ? 'Response' : response.definition.success,
     operationName,
   );
+
   const responseTypeImplementation = override.fetch
     .includeHttpResponseReturnType
     ? `export type ${responseTypeName} = {
-  data: ${response.definition.success || 'unknown'};
+  ${isNdJson ? `stream: Response // type: ${response.definition.success || 'unknown'}` : `data: ${response.definition.success || 'unknown'}`};
```

Comment:
I don't want to add type annotation comments because they are implicit and cause confusion, making it difficult to understand what is intended. If you want to know the contents of type, you should add a comment as a new line above the stream definition.

Reply:
Completely agree with @soartec-lab